libs/core/src/lib/shellbar/shellbar.component.scss

```diff
@@ -1,5 +1,11 @@
 @import 'fundamental-styles/dist/shellbar';
 
+.fd-shellbar--cx-side-nav {
+    box-shadow: var(--sapContent_Shadow0);
+    padding-left: 0.5rem;
+    z-index: 1;
+}
```

Comment:
is there a plan to move this to fund-styles or it's Angular specific?

Reply:
As discussed with @InnaAtanasova, this will stay here